README.md

```diff
@@ -1,42 +1,96 @@
-# Link Redux
+# Link-Redux
 *A Link to the Web*
 
 [![CircleCI](https://circleci.com/gh/fletcher91/link-redux.svg?style=svg)](https://circleci.com/gh/fletcher91/link-redux)
 [![Read the Docs](https://img.shields.io/readthedocs/pip.svg)](https://fletcher91.github.io/link-redux/)
 ![npm version](https://img.shields.io/npm/v/link-lib.svg)
 ![npm type definitions](https://img.shields.io/npm/types/link-redux.svg)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+
+Link-Redux (or Link) helps you render [Linked Data](https://ontola.io/what-is-linked-data/) (RDF) in React.
+Check out the [docs](https://fletcher91.github.io/link-redux/) and the [wiki](https://github.com/fletcher91/link-redux/wiki)!
+
+## Features
+- RDF data fetching & parsing, using [rdflib.js](https://github.com/linkeddata/rdflib.js/)
+- View management (error & loading handling, smart lookups, adapts to data)
+- Rule-based inference engine to determine which View is suitable
+- Actions / Data manipulation (in a flux pattern, like redux)
```

Comment:
It extends flux to be able to include the server in the loop 

More info here  https://github.com/fletcher91/link-lib/wiki/Hypermedia-API